src/lib/components/sections/navbar.svelte

```diff
@@ -165,22 +171,14 @@
   @media screen and (max-width: 900px) {
     nav {
       .container {
-        .logo-container {
-          width: unset;
-
-          .logo {
-            gap: 7px;
-
-            .badge {
-              height: 48px;
-            }
-          }
+        .logo-container .logo .badge {
+          height: 48px;
         }
 
         .pages {
           position: fixed;
           flex-direction: column;
-          justify-content: space-evenly;
+          justify-content: center;
```

Comment:
Justify center to condense the look of the buttons. Enough spacing like this already due to `gap: 64px`